packages/israel/README.md

```diff
@@ -0,0 +1,35 @@
+# 🇮🇳 Israel
+
+Contains all the states and union territories of israel:
+* Golan Heights
+* Hazafon
+* Hefa
+* Palestine
+* Hamerkaz
+* Tel-Aviv
+* Jerusalem
+* Hadarom
```

Comment:
Could [list the the locations in alphabetical order](https://github.com/VictorCazanave/svg-maps/blob/master/CONTRIBUTING.md#documentation) please?